base.css

```diff
@@ -12,7 +12,13 @@
     --white-page-background-color:#ffffffeb;
     --text-black-color:#333333;
     --text-white-color:#ffffffeb;
+    --text-grey-color:#605d5d;
     --grey-border-color:#9790908a;
+    --component-primary-color:#9edff5;
```

Comment:
A suggestion to change --component-primary-color to --primary-color, If you haven't used --primary-color in other places.

Reply:
Hi Abhijit,
          I am using primary color class for components library's theme color. The component-primary-color is used for blue which can be used by user as there primary color.so, can not change it to --primary-color.
Any other suggestions?

Reply:
Okay, I had read that we should use 3word classes less, If you are already using primary-color class then this is fine.